README.md

```diff
@@ -76,6 +76,63 @@ eval "$(pyenv virtualenv-init -)"
 
 After all, source `~/.bash_profile` and `~/.bashrc`.
 
+### Install nc
+
+To let ansible control the host itself and all the hadoop related containers, we need to install `nc` command:
+
+```bash
+yum install epel-release && yum install -y nc
+```
+
+Then configure the `~/.ssh/config` file in your host:
+
+```bash
+Host *
+    StrictHostKeyChecking no
+
+Host hadoop-master1
+    Hostname hadoop-master1.orb.local
+    User root
+    Port 22
+    ForwardAgent yes
+    IdentityFile /root/.ssh/id_rsa_hadoop_testing
+    ProxyCommand nc --proxy-type socks5 --proxy 127.0.0.1:18070 %h %p
+
+Host hadoop-worker1
+    Hostname hadoop-worker1.orb.local
+    User root
+    Port 22
+    ForwardAgent yes
+    IdentityFile /root/.ssh/id_rsa_hadoop_testing
+    ProxyCommand nc --proxy-type socks5 --proxy 127.0.0.1:18070 %h %p
+
+Host hadoop-worker2
+    Hostname hadoop-worker2.orb.local
+    User root
+    Port 22
+    ForwardAgent yes
+    IdentityFile /root/.ssh/id_rsa_hadoop_testing
+    ProxyCommand nc --proxy-type socks5 --proxy 127.0.0.1:18070 %h %p
+
+Host hadoop-worker3
+    Hostname hadoop-worker3.orb.local
+    User root
+    Port 22
+    ForwardAgent yes
+    IdentityFile /root/.ssh/id_rsa_hadoop_testing
+    ProxyCommand nc --proxy-type socks5 --proxy 127.0.0.1:18070 %h %p
```

Comment:
this does NOT work on my Ubuntu 22.04, I tune it to below instead

```
Host hadoop-*
    Hostname %h.orb.local
    User root
    Port 22
    ForwardAgent yes
    IdentityFile ~/.ssh/id_rsa_hadoop_testing
    StrictHostKeyChecking no
    ProxyCommand nc -x 127.0.0.1:18070 %h %p
```
The pattern `Host hadoop-*` maybe too wide, there is a chance that users have hosts that match the pattern, put this part at the end of `~/.ssh/config` for safe.

`IdentityFile /root/.ssh/id_rsa_hadoop_testing` we should not assume that user use root, `~/.ssh/id_rsa_hadoop_testing` should be used instead

`ProxyCommand nc --proxy-type socks5 --proxy 127.0.0.1:18070 %h %p` my `nc` does not support options `--proxy-socks5` and `--proxy`. I give an example tested on Ubuntu 22.04, let's provide a command that works at least on MacOS Ubuntu and CentOS. `127.0.0.1` should actually be the dockerd host, not a constant, though it does in the most time.